pkg/util/slice/slice_test.go

```diff
@@ -106,3 +106,48 @@ func TestContainsString(t *testing.T) {
 		t.Errorf("ContainsString didn't find the string by modifier")
 	}
 }
+
+func TestRemoveString(t *testing.T) {
+	modifier := func(s string) string {
+		if s == "ab" {
+			return "ee"
+		}
+		return s
+	}
+	tests := []struct {
+		input    []string
+		remove   string
+		modifier func(s string) string
+		want     []string
+	}{
+		{
+			input:    nil,
+			remove:   "",
+			modifier: nil,
+			want:     nil,
+		},
+		{
+			input:    []string{"a", "ab", "cdef"},
+			remove:   "",
+			modifier: nil,
+			want:     []string{"a", "ab", "cdef"},
+		},
+		{
```

Comment:
Could add a test case for:
* removing a string that doesn't exist
* removing duplicate strings in the input

Reply:
@msau42 
> removing a string that doesn't exist

Such test case was already present, but it wasn't obvious what's the purpose of the test case as it was removing an empty string.

> removing duplicate strings in the input

Added.

I also added names to test cases in order to describe the purpose of the individual test cases.